app/models/rdv.rb

```diff
@@ -370,6 +379,10 @@ def visio_url
     "https://webconf.numerique.gouv.fr/RdvServicePublic#{uuid}".gsub(/[-_]/, "")
   end
 
+  def busy_in_the_future?
```

Comment:
renommer la colonne en base aussi
```suggestion
  def not_cancelled_and_in_the_future?
```